pkg/provider/provider.go

```diff
@@ -43,10 +43,14 @@ type DeploymentResource struct {
 
 // NewDeploymentResource returns DeploymentResource with default values.
 func NewDeploymentResource() *DeploymentResource {
+	// Set NGINX_SERVICE_TYPE to LoadBalancer.
```

Comment:
I think we can get rid of this comment now as this is not telling us anything different from what the map declaration is telling. :)

Reply:
Okay